.rubocop.yml

```diff
@@ -50,3 +53,6 @@ RSpec/NestedGroups:
 
 RSpec/MultipleExpectations:
   Max: 10
+
+RSpec/ExampleLength:
+  Max: 10
```

Comment:
Same here. It might a sign that tests might need to be broken down.

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/7818449712a3ebb39bbdc369b1e88555cdf9ef4c

Reply:
This change was reverted based on [Oliver suggestions](https://github.com/Goose97/google-scraper-ruby/pull/20#discussion_r1280060741)